README.md

```diff
@@ -453,6 +465,9 @@ openHAB internally makes extensive use of the `java.time` package.
 openHAB-JS exports the excellent [JS-Joda](#https://js-joda.github.io/js-joda/) library via the `time` namespace, which is a native Javascript port of the same API standard used in Java for `java.time`.
 Anywhere that a native Java `ZonedDateTime` or `Duration` is required, the runtime will automatically convert a JS-Joda `ZonedDateTime` or `Duration` to its Java counterpart.
 
+The openHAB-provided [JS-Joda](#https://js-joda.github.io/js-joda/) library is extended with convenient functions relevant to openHAB usage.
+Those extensions are listed in [openHAB-specific extensions for JS-Joda](#openhab-specific-extensions-for-js-joda).
+
 Examples:

```

Comment:
Should we add an example of millisFromNow() here ?

Reply:
We could add it to the first block of examples.